--- 
title: "Nursing School Report"
author: "Steve Pittard"
date: "`r Sys.Date()`"
bibliography:
- book.bib
- packages.bib
description: This is a summary report of School of Nursing Faculty Interviews
documentclass: book
link-citations: yes
site: bookdown::bookdown_site
biblio-style: apalike
---

# Executive Summary and Recommendations

Conducting these interviews was a smooth process as those contacted were very forthcoming with helpful responses that were very consistent across the topic areas described in this report. Deviations of opinions did sometimes occur along the lines of junior and senior faculty although *all* of the senior faculty with whom I spoke were keenly aware of the difficulties facing junior faculty and appeared to be sympathetic. Everyone was collegial and supplied viewpoints that will benefit the School as a whole. The concerns expressed were entirely relevant to improving research processes and it's clear that even though there are some challenges, people are nonetheless working very hard to get things done.  In any case, the School of Nursing has definite strengths in its clinical programs combined with accomplished and talented faculty. It remains a priority to continue to apply for data-based grants and to this end the School must be able to point to “official” and strong support for analytics and computation within proposals.

## Participants

The participants in the interview process were: Jessica Wells, Irene Yang, Jinbing Bai, Erin Ferratti, Betsy Corwin, Nicole Carslon, Despina Tsementzi, Anna Knight, Deborah Bruner, Mary Gullate (to be conducted in mid-November). I also had a discussion with Karl Moran at Microbiome Insights since they performed work for Irene Yang. I also contacted Ethenia Why at the Biostatistics Core at Hopkins for comments on their role in Microbiome analysis. I've also got an inquiry to Steven Gill at the University of Rochester Microbiome Core. I've had extensive conversations with Madhusmita Behera who is the Director of Informatics at Winship Cancer Institute which is also working towards a usable solution for Data Management and Computation. To this end we have been speaking with both Seven Bridges Genomics and DNANexus.  

## Findings / Initial Recommendations:

Note that these represent suggested starting points which can flexibly be implemented. Most, if not all, of these recommendations are based on (or are motivated by) suggestions from faculty. Further discussion is warranted particularly on items relating to the formation of a central resource (1.2.2) and the licensing of a data management and computation framework (1.2.1). The following statements were derived from the more detailed sections (2-6) of this document which in turn were based on the faculty interviews. 

1. **Data management** is a very important aspect of analysis and research although the current practices for supporting these activities are less than ideal. Intermediate results and data transformations are not maintained alongside original data which impairs reproducibility. The data residing in Emory Box cannot be computed against and must be moved to individual computers (sometimes several) for downstream analysis. A framework is needed to maintain experimental data, code, and notebooks. **The School should consider use of a tool such as DNANexus or Seven Bridges Genomics either of which would provide comprehensive support for managing sequencing-based projects, analytics, as well as the convenient addition of self-developed pipelines.** This would address a number of concerns of having "everything in one place" (to the extent that it is possible) along with computational results in a format that would enable reproducibility. While RedCap is useful for maintaining study information, being able to link in sequencing and sample information can be challenging. 

2. **Central Nursing Analytics 'Omics' Resource** Availability of professionals who are well-versed in analytics, statistics, and Omics are critical to the research mission of the School. **To this end, the formation of a central service facility *within* the School is recommended**. This entity would provide services for analytics, computing, and interactions with sequencing facilities (where desired) and basically exist for the benefit of investigators. Duties would involve assistance with data management, cleaning, preliminary analysis, and ongoing participation in projects. Organizing training and providing orientation to the framework mentioned in item #1 would also be on the menu of services. It is important to note that such a resource could be referenced in applications for grants involving large scale processing. This would be evidence of a formal institutional commitment to Nursing analytics. Opinions vary as to the exact composition of this group and/or if it should draw from existing personnel (who aren't already fully funded) but this is a strongly supported idea among those interviewed. For a more detailed analysis of this topic please consult section 4) Interaction with Cores. 

3. **Improved Project Estimates** Researchers would like more detailed upfront estimates of statistics, analytics, and computation costs to better estimate budgets. Not knowing how much (in time, money, and personnel) the key aspects of a project will "really" involve is a concern. To this end, having access to a person familiar with the aforementioned areas to help with up front assessment would be very useful. This could be implemented separately of option #2 though if the resource in option #2 did exist then this person could also be part of that that. The overall idea is to have accurate and reliable estimates of sequencing, analytics, and computation work. Some junior faculty feel that they cannot fully access personnel due to lack of funding. 

4. **Computation** remains a challenge in that the School does not currently "own" any significant computational resources which has led researchers to leverage personal and external professional relationships to accomplish work. This approach 1) does not scale well, 2) works against the idea of reproducible research, and 3) impairs collaborative ability.  While Amazon Web Services remains a possibility, it requires significant training and assistance in setting up, maintaining, and breaking down projects which makes it unlikely that individual investigators could conveniently use such a service in absence of ongoing help. While direct use of Amazon does remain an option, the School should consider the use of a tool such as DNANexus or Seven Bridges Genomics that overlays Amazon AWS while providing intuitive support for data management and computation. **Solving the data management and computation problem, in addition to being able to easily execute and add pipelines, is a more direct route to productivity than building out infrastructure in the cloud or locally**.

  + A secondary approach might involve partnering with the Department of Biomedical Informatics which maintains a sophisticated, well-functioning, well-supported computational cluster dedicated to aggressive research computing. This would have to be arranged but none of the projects currently being considered within the School of Nursing would represent a challenge to that environment. BMI would be an appropriate partner because research computing is "baked-in" to their mission and is thus seen as an essential service for researchers. I discussed a potential relationship with Jim Kinney (BMI System Lead) earlier this year but would need to renew those talks. 
 
5. **Alternatives to Local Processing** The School should develop relationships with trusted external scientific resource providers to process urgent requests for sample processing and analysis consultation. This should also include training resources which could be accomplished virtually, on-premise (for larger groups), or remotely. Not every project will require this but there should be a reliable outlet for "blocked" projects or those requiring a rapid turnaround. In particular, we should organize training to learn "best practices" in the processing and analysis of Microbiome and Metabolomics. At least one faculty member has used an external lab for Microbiome and another is contemplaing the resequencing of a large number of samples via the Ravel Lab at the Institute of Genome Sciences to obtain domain expertise starting at the sequencing level.  

6. Improving general **software literacy** is very important as is being able to import/transform data, accomplish analytic tasks, create plots, query databases, and create digital assets will facilitate publication and grant submissions. Many faculty pursue self-education but are interested in becoming much more self-sufficient with open source tools and the UNIX command line. This can be accomplished via participation in the Big Data course or more specific classes. However, one excellent (and cheap) resource is to arrange for an onsite [Software Carpentry](https://software-carpentry.org/) session which is a 1 to 3 day workshop devoted to teaching basic lab skills for research computing. These sessions are professionally taught and include hands on labs to learn the UNIX command line, R and Python Programming, Git, SQL and Databases. These are targeted to the novice but would also serve to reinforce skills. The material is open source and maintained on GitHub so we could possibly offer the course material with local teaching resources. The following graph illustrates the motovation levels for various software and command line topics after taking a Core level Software Carpentry class. 

![](./PICS/carpentry.png){width=550}

7. **Student Recruitment** - Deliberate efforts should be made to recruit graduate students with ability and/or interest in data manipulation and analysis since the nature of research within the school will require such a background. While no one beleives that using **only** student "labor" to accomplish projects is reasonable there is a goal of having software literate students who can "dive in" to projects. 

The following sections (2-6) represent the interview summary information which was used as a basis for the above recommendations. Each section takes a deeper dive into the area under consideration.  