<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Nursing School Research Analytics and Computation Report</title>
  <meta name="description" content="This is a summary report of School of Nursing Faculty Interviews">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Nursing School Research Analytics and Computation Report" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a summary report of School of Nursing Faculty Interviews" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Nursing School Research Analytics and Computation Report" />
  
  <meta name="twitter:description" content="This is a summary report of School of Nursing Faculty Interviews" />
  

<meta name="author" content="Steve Pittard">


<meta name="date" content="2018-12-04">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  

<link rel="next" href="data-management.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Nell Hodgson Nursing School</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Executive Summary and Recommendations</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#participants"><i class="fa fa-check"></i><b>1.1</b> Participants</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#motivations"><i class="fa fa-check"></i><b>1.2</b> Motivations</a><ul>
<li class="chapter" data-level="1.2.1" data-path="index.html"><a href="index.html#the-omics-problem"><i class="fa fa-check"></i><b>1.2.1</b> The “Omics” Problem</a></li>
<li class="chapter" data-level="1.2.2" data-path="index.html"><a href="index.html#proactive-planning-and-secondary-analysis"><i class="fa fa-check"></i><b>1.2.2</b> Proactive Planning and Secondary Analysis</a></li>
<li class="chapter" data-level="1.2.3" data-path="index.html"><a href="index.html#data-access-and-collaboration"><i class="fa fa-check"></i><b>1.2.3</b> Data Access and Collaboration</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#findings-initial-recommendations"><i class="fa fa-check"></i><b>1.3</b> Findings / Initial Recommendations:</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="data-management.html"><a href="data-management.html"><i class="fa fa-check"></i><b>2</b> Data Management</a><ul>
<li class="chapter" data-level="2.1" data-path="data-management.html"><a href="data-management.html#data-hosting"><i class="fa fa-check"></i><b>2.1</b> Data Hosting</a></li>
<li class="chapter" data-level="2.2" data-path="data-management.html"><a href="data-management.html#source-code-maintenance"><i class="fa fa-check"></i><b>2.2</b> Source Code Maintenance</a></li>
<li class="chapter" data-level="2.3" data-path="data-management.html"><a href="data-management.html#notebooks-and-reproducible-research"><i class="fa fa-check"></i><b>2.3</b> Notebooks and Reproducible Research</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="software-literacy.html"><a href="software-literacy.html"><i class="fa fa-check"></i><b>3</b> Software Literacy</a><ul>
<li class="chapter" data-level="3.1" data-path="software-literacy.html"><a href="software-literacy.html#concerns-about-falling-behind"><i class="fa fa-check"></i><b>3.1</b> Concerns About Falling Behind</a></li>
<li class="chapter" data-level="3.2" data-path="software-literacy.html"><a href="software-literacy.html#workshops"><i class="fa fa-check"></i><b>3.2</b> Workshops</a></li>
<li class="chapter" data-level="3.3" data-path="software-literacy.html"><a href="software-literacy.html#sasspss-vs-open-source-tools"><i class="fa fa-check"></i><b>3.3</b> SAS/SPSS vs Open Source Tools</a></li>
<li class="chapter" data-level="3.4" data-path="software-literacy.html"><a href="software-literacy.html#attracting-software-literate-students"><i class="fa fa-check"></i><b>3.4</b> Attracting Software Literate Students</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="interaction-with-cores.html"><a href="interaction-with-cores.html"><i class="fa fa-check"></i><b>4</b> Interaction with Cores</a><ul>
<li class="chapter" data-level="4.1" data-path="interaction-with-cores.html"><a href="interaction-with-cores.html#expectations-of-cores"><i class="fa fa-check"></i><b>4.1</b> Expectations of Cores</a></li>
<li class="chapter" data-level="4.2" data-path="interaction-with-cores.html"><a href="interaction-with-cores.html#understanding-results"><i class="fa fa-check"></i><b>4.2</b> Understanding Results</a></li>
<li class="chapter" data-level="4.3" data-path="interaction-with-cores.html"><a href="interaction-with-cores.html#need-for-more-consultation"><i class="fa fa-check"></i><b>4.3</b> Need for More Consultation</a></li>
<li class="chapter" data-level="4.4" data-path="interaction-with-cores.html"><a href="interaction-with-cores.html#possible-solutions"><i class="fa fa-check"></i><b>4.4</b> Possible Solutions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="project-initiation-dynamics.html"><a href="project-initiation-dynamics.html"><i class="fa fa-check"></i><b>5</b> Project Initiation Dynamics</a><ul>
<li class="chapter" data-level="5.1" data-path="project-initiation-dynamics.html"><a href="project-initiation-dynamics.html#continuity-throughout-the-research-process"><i class="fa fa-check"></i><b>5.1</b> Continuity Throughout the Research Process</a></li>
<li class="chapter" data-level="5.2" data-path="project-initiation-dynamics.html"><a href="project-initiation-dynamics.html#estimating-effort"><i class="fa fa-check"></i><b>5.2</b> Estimating Effort</a></li>
<li class="chapter" data-level="5.3" data-path="project-initiation-dynamics.html"><a href="project-initiation-dynamics.html#grants"><i class="fa fa-check"></i><b>5.3</b> Grants</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="computation.html"><a href="computation.html"><i class="fa fa-check"></i><b>6</b> Computation</a><ul>
<li class="chapter" data-level="6.1" data-path="computation.html"><a href="computation.html#data-management-and-computation"><i class="fa fa-check"></i><b>6.1</b> Data Management and Computation</a></li>
<li class="chapter" data-level="6.2" data-path="computation.html"><a href="computation.html#cloud-vs-local-resources"><i class="fa fa-check"></i><b>6.2</b> Cloud vs Local Resources</a></li>
<li class="chapter" data-level="6.3" data-path="computation.html"><a href="computation.html#desktop-support-for-open-source-analysis-tools"><i class="fa fa-check"></i><b>6.3</b> Desktop Support for Open Source Analysis Tools</a></li>
<li class="chapter" data-level="6.4" data-path="computation.html"><a href="computation.html#general-it-support"><i class="fa fa-check"></i><b>6.4</b> General IT Support</a></li>
<li class="chapter" data-level="6.5" data-path="computation.html"><a href="computation.html#databases-and-applications"><i class="fa fa-check"></i><b>6.5</b> Databases and Applications</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Nursing School Research Analytics and Computation Report</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Nursing School Research Analytics and Computation Report</h1>
<h4 class="author"><em>Steve Pittard</em></h4>
<h4 class="date"><em>2018-12-04</em></h4>
</div>
<div id="executive-summary-and-recommendations" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Executive Summary and Recommendations</h1>
<p>Conducting these interviews was a smooth process as those contacted were very forthcoming with helpful responses that were very consistent across the topic areas described in this report. Deviations of opinions did sometimes occur along the lines of junior and senior faculty although all of the senior faculty with whom I spoke were keenly aware of the difficulties facing junior faculty and appeared to be sympathetic. Everyone was collegial and supplied viewpoints that will benefit the School as a whole. The concerns expressed were entirely relevant to improving research processes and it’s clear that even though there are some challenges, people are nonetheless working very hard to get things done. The School of Nursing has definite strengths in its clinical programs combined with accomplished and talented faculty. It remains a priority to continue to apply for data-based grants and to this end the School must be able to point to official and strong support for analytics and computation within proposals to be competitive.</p>
<div id="participants" class="section level2">
<h2><span class="header-section-number">1.1</span> Participants</h2>
<p>The participants in the interview process were: Linda McCauley, Jessica Wells, Irene Yang, Rebecca Mitchell, Jinbing Bai, Erin Ferratti, Betsy Corwin, Nicole Carslon, Despina Tsementzi, Anna Knight, Deborah Bruner, Mary Gullate, Melinda Higgins, and Jason Atkins. I also had a discussion with Karl Moran at Microbiome Insights since they performed work for Irene Yang. I also contacted Ethenia Why at the Biostatistics Core at Hopkins for comments on their role in Microbiome analysis. I’ve also got an inquiry to Steven Gill at the University of Rochester Microbiome Core. I’ve had conversations with Madhusmita Behera who is the Director of Informatics at Winship Cancer Institute which is also working towards a usable solution for Data Management and Computation.</p>
</div>
<div id="motivations" class="section level2">
<h2><span class="header-section-number">1.2</span> Motivations</h2>
<div id="the-omics-problem" class="section level3">
<h3><span class="header-section-number">1.2.1</span> The “Omics” Problem</h3>
<p>The magnitude and highly dynamic nature of omics-based projects can lead to interesting explorations though ultimately the research should lead to publications in high impact journals and successful grant applications. These goals, however, can be frustrated by uncertainty about how best to manage data, which analytic techniques are most appropriate, where the computation should be performed, and the how to effecgtively determine the biological relevance of the result. This can also be particularly challenging for doctoral students who can sometimes receive conflicting advice on how to structure omics-based thesis projects. Creating smooth workflows using omics-data is essential to progress and publications thus startegies must be developed to effectively deal with this situation.</p>
<p>At a functional level, the development and implementation of a common logistical workflow would be helpful to guide doctoral students and faculty throughout the project lifecycle. This is particularly true in the planning phase wherein documenting required personnel, appropriate percent effort, and anticipated service core involvement becomes essential to the outcome of having a defensible, publishable result that is accomplished according to known standards for a given Omics type (e.g. microbiome, metabolomics).</p>
</div>
<div id="proactive-planning-and-secondary-analysis" class="section level3">
<h3><span class="header-section-number">1.2.2</span> Proactive Planning and Secondary Analysis</h3>
<p>There is the impression that many current omics-based projects are more reactive than proactive which then leads to a “best effort in the moment” approach to completing the work that can result in a sub optimal outcome. Some of this relates to user uncertainty with results being returned by supporting Core facilities which represent just a first step towards an improved comprehension of the experiment. Additional study with various bioinformatics and analysis tools will always be required which, in turn, presumes a base level of literacy and facility with programming languages such as R, Python, or similar packages.</p>
<p>The following figure shows a typical work flow based on a gut microbiome sample. The customary approach has been to accomplish Steps 2-3 via a trusted service core facility with steps 1, 4 and 5 being performed locally within the school by faculty, students, professional staff, collaborators and/or a combination thereof. Some faculty have leveraged personal relationships and employed external labs to accomplish all steps simply to expedite project completion although being able to conduct the analysis locally and within a reasonable time frame is a goal.</p>
<div class="figure" style="text-align: center"><span id="fig:sdia"></span>
<img src="PICS/sdia.png" alt="Typical Microbiome Workflow" width="60%" />
<p class="caption">
Figure 1.1: Typical Microbiome Workflow
</p>
</div>
<p>From the Core point of view the priority is to execute a discrete , high-quality experiment and in the case of Microbiome projects, limit batch effects by employing informed plating strategies. So, an ongoing bioinformatics consultation might not be possible without additional financial support although this does not imply the existence of resident domain experts waiting to be engaged. In any case, closing the gap between user expectations and the typical Core product is a concern as is having reasonable access to expertise (internal or otherwise).</p>
<p>If the analysis steps are broken down into the component tools (e.g. Qiime, DADA2, Mothur, etc) there is wide variation in software knowledge on how to best implement the software to arrive at a biologically relevant result. Additional experimentation with various softwware tools and pipelines workflow will occur although some tools require more substantial computation environments such as the Amazon cloud which itself can present a steep learning curve for investigators. After receiving initial results from the Core, investigators will frequently reengage with followup questions to better understand the result. Not all Cores have the resident domain expertise to sustain and ongoing followup dialogue or, if they do, staff might not currently be available.</p>
</div>
<div id="data-access-and-collaboration" class="section level3">
<h3><span class="header-section-number">1.2.3</span> Data Access and Collaboration</h3>
<p>A more fundamental problem exists in having convenient access to data as it makes its way through the analytics pipeline and in understanding what transformations have been applied (or not) on the way to a candidate final result. A solution such as RedCap is important though it can host only the more structured part of the project whereas the associated omics data sets typically wind up on an Emory Box account that cannot be computed against. This leads to data being distributed to colleagues resulting in multiple copies that are transformed according to the needs of whomever is currently working on it (statistician, biologist, bioinformatician). Unfortuntately, the transformed data and results (as well as the code used to produce it) are rarely reintegrated back into a repository for common reference.</p>
<p>This is understandable in that there is not a common platform upon which to perform the work which could also serve as a sharing / collaboration environment. It is generally agreed that maintaining projects on laptops is a problem though it is a popular approach. However, not all PIs feel comfortable sharing data especially in scenarios where the data has been difficult to obtain. It’s agreed that research reproducibility is important but there are sensitivities to having data commonly available during the project. (Appropriate security approaches can help here). Independently of these concerns, the funding agencies, (via concepts such as the <a href="https://www.force11.org/group/fairgroup/fairprinciples">“FAIR Guiding Principles”</a> for data management and stewardship), are very much concerned with the “long-term care” of valuable digital assets, thus the appropriate annotation and archival of data is an obligation.</p>
</div>
</div>
<div id="findings-initial-recommendations" class="section level2">
<h2><span class="header-section-number">1.3</span> Findings / Initial Recommendations:</h2>
<p>Note that these represent suggested starting points which can flexibly be implemented. Most, if not all, of these recommendations are based on (or are motivated by) suggestions from faculty who have attempted to address these problems in various ways including leverating personal, professional, and external relationships. Further discussion is warranted particularly on items relating to the formation of a central resource (1.2.2) and the licensing of a data management and computation framework (1.2.1). The following statements were derived from the more detailed sections (2-6) of this document which in turn were based on the faculty interviews.</p>
<ol style="list-style-type: decimal">
<li><strong>Data management</strong> is a very important aspect of analysis and research although the current practices for supporting these activities are less than ideal. Intermediate results and data transformations are not maintained alongside original data which impairs reproducibility. The data residing in Emory Box cannot be computed against and must be moved to individual computers (sometimes several) for downstream analysis. A framework is needed to maintain experimental data, code, and notebooks. <strong>The School should consider use of a tool such as DNANexus or Seven Bridges Genomics either of which would provide comprehensive support for managing sequencing-based projects, analytics, as well as the convenient addition of self-developed pipelines.</strong></li>
</ol>
<ul>
<li>This would address a number of concerns of having “everything in one place” (to the extent that it is possible) along with computational results in a format that would enable reproducibility. Additionally, a solution such as DNANexus would facilitate the integration of genomic data with clinical and other phenotypic data in a secure and compliant environment. While RedCap is useful for maintaining study information, being able to link in sequencing and sample information can be challenging. For a more detailed analysis of this topic please consult section 4) Interaction with Cores.</li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li><p><strong>Nursing-Centric Analytics ‘Omics’ Resource</strong> Availability of professionals who are well-versed in analytics, statistics, and Omics are critical to the research mission of the School. <strong>To this end, the formation of a nursing-centric service facility primarily for the School is recommended</strong>. This entity would provide services for analytics, computing, and interactions with sequencing facilities (where desired) and generally exist for the benefit of investigators. Whether this service center is an “official university core” or a formalized group is up for debate although the feedback on this topic is clear in that the group should priortize Nursing School interests while also being available for salary support via external projects albeit secondarily. Such a group would not replace or diminish the relationships with existing University Cores but rather improve the working dynamic since Nursing investigators would have an adovcate acting on their behalf. Duties would involve assistance with data management, cleaning, preliminary analysis, and ongoing participation in projects. Organizing training and providing orientation to the framework mentioned in item #1 would also be on the menu of services. It is important to note that such a resource could be referenced in applications for grants involving large scale processing. This would be evidence of a formal institutional commitment to Nursing analytics. Opinions vary as to the exact composition of this group and/or if it should draw from existing personnel (who aren’t already fully funded) but this is a strongly supported idea among those interviewed. For a more detailed analysis of this topic please consult section 4) Interaction with Cores.</p></li>
<li><p><strong>Improved Project Logistics</strong> Researchers would like more detailed upfront estimates of statistics, analytics, and computation costs to better project cost of staff involvement. Not knowing how much (in time, money, and personnel) the key aspects of a project will “really” involve is a concern. To this end, having access to a person familiar with the aforementioned areas to help with up front assessment would be very useful. This could be implemented separately of option #2 though if the resource in option #2 did exist then this person could also be part of that that. The overall idea is to have accurate and reliable estimates of sequencing, analytics, and computation work. Some junior faculty feel that they cannot fully access personnel due to lack of funding. In general, having a standard logistical workflow that specifies who to involve, when, and to what extent is needed. This includes collaborating domain expertise (statistics and bioinformatics personnel) as well as key Core facilities - internal or external to Emory.</p></li>
<li><p><strong>Computation</strong> remains a challenge in that the School does not currently “own” any significant computational resources which has led researchers to leverage personal and external professional relationships to accomplish work. This approach 1) does not scale well, 2) works against the idea of reproducible research, and 3) impairs collaborative ability. While Amazon Web Services remains a possibility, it requires significant training and assistance in setting up, maintaining, and breaking down projects which makes it unlikely that individual investigators could conveniently use such a service in absence of ongoing help. While direct use of Amazon does remain an option, the School should consider the use of a tool such as DNANexus or Seven Bridges Genomics that overlays Amazon AWS while providing intuitive support for data management and computation. <strong>Solving the data management and computation problem, in addition to being able to easily execute and add pipelines, is a more direct route to productivity than building out infrastructure in the cloud or locally</strong>.</p></li>
</ol>
<ul>
<li>A secondary approach might involve partnering with the Department of Biomedical Informatics which maintains a sophisticated, well-functioning, well-supported computational cluster dedicated to aggressive research computing. This would have to be arranged but none of the projects currently being considered within the School of Nursing would represent a challenge to that environment. BMI would be an appropriate partner because research computing is “baked-in” to their mission and is thus seen as an essential service for researchers. I discussed a potential relationship with Jim Kinney (BMI System Lead) earlier this year but would need to renew those talks.</li>
</ul>
<ol start="5" style="list-style-type: decimal">
<li><p><strong>Alternatives to Local Sample Processing</strong> The School should develop relationships with trusted external scientific resource providers to process urgent requests for sample processing and analysis consultation. This should also include training resources which could be accomplished virtually, on-premise (for larger groups), or remotely. Not every project will require this but there should be a reliable outlet for “blocked” projects or those requiring a rapid turnaround. In particular, we should organize training to learn “best practices” in the processing and analysis of Microbiome and Metabolomics. At least one faculty member has used an external lab for Microbiome and another is contemplaing the resequencing of a large number of samples via the Ravel Lab at the Institute of Genome Sciences to obtain domain expertise starting at the sequencing level.</p></li>
<li><p>Improving general <strong>software literacy</strong> is very important as is being able to import/transform data, accomplish analytic tasks, create plots, query databases, and create digital assets will facilitate publication and grant submissions. Many faculty pursue self-education but are interested in becoming much more self-sufficient with open source tools and the UNIX command line. This can be accomplished via participation in the Big Data course or more specific classes. However, one excellent (and cheap) resource is to arrange for an onsite <a href="https://software-carpentry.org/">Software Carpentry</a> session which is a 1 to 3 day workshop devoted to teaching basic lab skills for research computing. These sessions are professionally taught and include hands on labs to learn the UNIX command line, R and Python Programming, Git, SQL and Databases. These are targeted to the novice but would also serve to reinforce skills. The material is open source and maintained on GitHub so we could possibly offer the course material with local teaching resources. The following graph illustrates the motovation levels for various software and command line topics after taking a Core level Software Carpentry class.</p></li>
</ol>
<div class="figure">
<img src="PICS/carpentry.png" width="550" />

</div>
<ol start="7" style="list-style-type: decimal">
<li><strong>Student Recruitment</strong> - Deliberate efforts should be made to recruit graduate students with ability and/or interest in data manipulation and analysis since the nature of research within the school will require such a background. While no one beleives that using <strong>only</strong> student “labor” to accomplish projects is reasonable, there is a goal of having software literate students who can “dive in” to projects. However, the School Dean did express a specific concern that it can be challenging for doctoral students who can sometimes receive conflicting advice on how to structure omics-based thesis projects. So while having software savvy students is a plus, they will still require guidance in the selection and execution of their thesis work which in turn assumes the existence of a knowledgeable community.</li>
</ol>
<p>The following sections (2-6) represent the interview summary information which was used as a basis for the above recommendations. Each section takes a deeper dive into the area under consideration.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="data-management.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": ["Nursing Report.pdf", "Nursing Report.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
