<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Nursing School Research Analytics and Computation Report</title>
  <meta name="description" content="This is a summary report of School of Nursing Faculty Interviews">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Nursing School Research Analytics and Computation Report" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a summary report of School of Nursing Faculty Interviews" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Nursing School Research Analytics and Computation Report" />
  
  <meta name="twitter:description" content="This is a summary report of School of Nursing Faculty Interviews" />
  

<meta name="author" content="Steve Pittard">


<meta name="date" content="2019-01-23">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  

<link rel="next" href="data-management-1.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Nell Hodgson Nursing School</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Motivations</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#background"><i class="fa fa-check"></i><b>1.1</b> Background</a><ul>
<li class="chapter" data-level="1.1.1" data-path="index.html"><a href="index.html#the-omics-problem"><i class="fa fa-check"></i><b>1.1.1</b> The “Omics” Problem</a></li>
<li class="chapter" data-level="1.1.2" data-path="index.html"><a href="index.html#the-knowledge-gap---communication-with-cores"><i class="fa fa-check"></i><b>1.1.2</b> The Knowledge Gap - Communication With Cores</a></li>
<li class="chapter" data-level="1.1.3" data-path="index.html"><a href="index.html#data-access-and-collaboration"><i class="fa fa-check"></i><b>1.1.3</b> Data Access and Collaboration</a></li>
<li class="chapter" data-level="1.1.4" data-path="index.html"><a href="index.html#the-promise-of-cloud-computing"><i class="fa fa-check"></i><b>1.1.4</b> The Promise of Cloud Computing</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#executive-summary-and-initial-recommendations"><i class="fa fa-check"></i><b>1.2</b> Executive Summary and Initial Recommendations:</a><ul>
<li class="chapter" data-level="1.2.1" data-path="index.html"><a href="index.html#data-management"><i class="fa fa-check"></i><b>1.2.1</b> <strong>Data management</strong></a></li>
<li class="chapter" data-level="1.2.2" data-path="index.html"><a href="index.html#nursing-centric-analytics-omics-resource"><i class="fa fa-check"></i><b>1.2.2</b> <strong>Nursing-Centric Analytics ‘Omics’ Resource</strong></a></li>
<li class="chapter" data-level="1.2.3" data-path="index.html"><a href="index.html#improved-project-logistics"><i class="fa fa-check"></i><b>1.2.3</b> <strong>Improved Project Logistics</strong></a></li>
<li class="chapter" data-level="1.2.4" data-path="index.html"><a href="index.html#research-desktop-support"><i class="fa fa-check"></i><b>1.2.4</b> <strong>Research Desktop Support</strong></a></li>
<li class="chapter" data-level="1.2.5" data-path="index.html"><a href="index.html#high-performance-computing"><i class="fa fa-check"></i><b>1.2.5</b> <strong>High Performance Computing</strong></a></li>
<li class="chapter" data-level="1.2.6" data-path="index.html"><a href="index.html#alternatives-to-local-core-processing"><i class="fa fa-check"></i><b>1.2.6</b> <strong>Alternatives to Local Core Processing</strong></a></li>
<li class="chapter" data-level="1.2.7" data-path="index.html"><a href="index.html#software-literacy"><i class="fa fa-check"></i><b>1.2.7</b> <strong>Software Literacy</strong></a></li>
<li class="chapter" data-level="1.2.8" data-path="index.html"><a href="index.html#student-recruitment"><i class="fa fa-check"></i><b>1.2.8</b> <strong>Student Recruitment</strong></a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#the-interview-process"><i class="fa fa-check"></i><b>1.3</b> The Interview Process</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="data-management-1.html"><a href="data-management-1.html"><i class="fa fa-check"></i><b>2</b> Data Management</a><ul>
<li class="chapter" data-level="2.1" data-path="data-management-1.html"><a href="data-management-1.html#data-hosting"><i class="fa fa-check"></i><b>2.1</b> Data Hosting</a></li>
<li class="chapter" data-level="2.2" data-path="data-management-1.html"><a href="data-management-1.html#source-code-maintenance"><i class="fa fa-check"></i><b>2.2</b> Source Code Maintenance</a></li>
<li class="chapter" data-level="2.3" data-path="data-management-1.html"><a href="data-management-1.html#notebooks-and-reproducible-research"><i class="fa fa-check"></i><b>2.3</b> Notebooks and Reproducible Research</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="software-literacy-1.html"><a href="software-literacy-1.html"><i class="fa fa-check"></i><b>3</b> Software Literacy</a><ul>
<li class="chapter" data-level="3.1" data-path="software-literacy-1.html"><a href="software-literacy-1.html#concerns-about-falling-behind"><i class="fa fa-check"></i><b>3.1</b> Concerns About Falling Behind</a></li>
<li class="chapter" data-level="3.2" data-path="software-literacy-1.html"><a href="software-literacy-1.html#workshops"><i class="fa fa-check"></i><b>3.2</b> Workshops</a></li>
<li class="chapter" data-level="3.3" data-path="software-literacy-1.html"><a href="software-literacy-1.html#sasspss-vs-open-source-tools"><i class="fa fa-check"></i><b>3.3</b> SAS/SPSS vs Open Source Tools</a></li>
<li class="chapter" data-level="3.4" data-path="software-literacy-1.html"><a href="software-literacy-1.html#attracting-software-literate-students"><i class="fa fa-check"></i><b>3.4</b> Attracting Software Literate Students</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="interaction-with-cores.html"><a href="interaction-with-cores.html"><i class="fa fa-check"></i><b>4</b> Interaction with Cores</a><ul>
<li class="chapter" data-level="4.1" data-path="interaction-with-cores.html"><a href="interaction-with-cores.html#expectations-of-cores"><i class="fa fa-check"></i><b>4.1</b> Expectations of Cores</a></li>
<li class="chapter" data-level="4.2" data-path="interaction-with-cores.html"><a href="interaction-with-cores.html#understanding-results"><i class="fa fa-check"></i><b>4.2</b> Understanding Results</a></li>
<li class="chapter" data-level="4.3" data-path="interaction-with-cores.html"><a href="interaction-with-cores.html#need-for-more-consultation"><i class="fa fa-check"></i><b>4.3</b> Need for More Consultation</a></li>
<li class="chapter" data-level="4.4" data-path="interaction-with-cores.html"><a href="interaction-with-cores.html#possible-solutions"><i class="fa fa-check"></i><b>4.4</b> Possible Solutions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="project-initiation-dynamics.html"><a href="project-initiation-dynamics.html"><i class="fa fa-check"></i><b>5</b> Project Initiation Dynamics</a><ul>
<li class="chapter" data-level="5.1" data-path="project-initiation-dynamics.html"><a href="project-initiation-dynamics.html#continuity-throughout-the-research-process"><i class="fa fa-check"></i><b>5.1</b> Continuity Throughout the Research Process</a></li>
<li class="chapter" data-level="5.2" data-path="project-initiation-dynamics.html"><a href="project-initiation-dynamics.html#estimating-effort"><i class="fa fa-check"></i><b>5.2</b> Estimating Effort</a></li>
<li class="chapter" data-level="5.3" data-path="project-initiation-dynamics.html"><a href="project-initiation-dynamics.html#grants"><i class="fa fa-check"></i><b>5.3</b> Grants</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="computation.html"><a href="computation.html"><i class="fa fa-check"></i><b>6</b> Computation</a><ul>
<li class="chapter" data-level="6.1" data-path="computation.html"><a href="computation.html#data-management-and-computation"><i class="fa fa-check"></i><b>6.1</b> Data Management and Computation</a></li>
<li class="chapter" data-level="6.2" data-path="computation.html"><a href="computation.html#cloud-vs-local-resources"><i class="fa fa-check"></i><b>6.2</b> Cloud vs Local Resources</a><ul>
<li class="chapter" data-level="6.2.1" data-path="computation.html"><a href="computation.html#onboarding-cloud-support"><i class="fa fa-check"></i><b>6.2.1</b> Onboarding Cloud Support</a></li>
<li class="chapter" data-level="6.2.2" data-path="computation.html"><a href="computation.html#post-project-data-archival"><i class="fa fa-check"></i><b>6.2.2</b> Post Project Data Archival</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="computation.html"><a href="computation.html#desktop-support-for-open-source-analysis-tools"><i class="fa fa-check"></i><b>6.3</b> Desktop Support for Open Source Analysis Tools</a></li>
<li class="chapter" data-level="6.4" data-path="computation.html"><a href="computation.html#general-it-support"><i class="fa fa-check"></i><b>6.4</b> General IT Support</a></li>
<li class="chapter" data-level="6.5" data-path="computation.html"><a href="computation.html#databases-and-applications"><i class="fa fa-check"></i><b>6.5</b> Databases and Applications</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Nursing School Research Analytics and Computation Report</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Nursing School Research Analytics and Computation Report</h1>
<h4 class="author"><em>Steve Pittard</em></h4>
<h4 class="date"><em>2019-01-23</em></h4>
</div>
<div id="motivations" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Motivations</h1>
<div id="background" class="section level2">
<h2><span class="header-section-number">1.1</span> Background</h2>
<p>The original scope of this report was focused primarily on problems associated with managing data and conducting omics-based research though it rapidly became apparent that there was as much concern about getting more support from Emory Core facilities. To a lesser extent (although still important) there is frustration with not being able to conveniently obtain data from the Clinical Data Warehouse and the perceived lack of customer focus exhibited by the Health/Research Services personnel. While data security is important, it must be balanced with the needs of the School and institution at large. Research Services personnel should work enthusiastically to enable projects to the extent they are possible within the stated security limitations. Moreover, the staff should have knowledge and experience about the biomedical research process and work to identify quicker paths to obtaining and organizing data for downstream research.</p>
<div id="the-omics-problem" class="section level3">
<h3><span class="header-section-number">1.1.1</span> The “Omics” Problem</h3>
<p>The magnitude and highly dynamic nature of omics-based projects can lead to interesting explorations though ultimately the research should lead to publications in high impact journals and successful grant applications. These goals, however, can be frustrated by uncertainty about how best to manage data, which analytic techniques are most appropriate, where the computation should be performed, and how to effectively determine the biological relevance of the result. This can also be particularly challenging for doctoral students who can sometimes receive conflicting advice on how to structure omics-based thesis projects. Creating smooth workflows using omics-data is essential to progress and publications thus strategies must be developed to effectively deal with this situation.</p>
<p>At a functional level, the development and implementation of a common logistical workflow would be helpful to guide faculty and doctoral students throughout the project life cycle. This is particularly true in the planning phase wherein documenting required personnel, appropriate percent effort, and anticipated service core involvement becomes essential to the outcome of having a defensible, publishable result that is accomplished according to known standards for a given Omics type (e.g. microbiome, metabolomics).</p>
</div>
<div id="the-knowledge-gap---communication-with-cores" class="section level3">
<h3><span class="header-section-number">1.1.2</span> The Knowledge Gap - Communication With Cores</h3>
<p>Nursing School investigators engage Emory Core facilities to obtain services not commonly available or affordable within individual labs. The expectation is that these engagements will advance the quality and speed of research and that results emanating from these centers will be informative, accurate, and reflect a high level of consideration for individual project goals. <strong>However, a majority of those interviewed have concerns about the dynamics and products being returned from the Cores.</strong> (A notable exception is the Lipidomics facility which demonstrates an enthusiastic level of interest for users). The general concerns relate to up front project design, the interpetability of the results, as well as the overall level of attention given to the project. The impression is that many current omics-based projects wind up being unnecessarily reactive and delayed since investigators (and students) have ongoing questions after receiving results when, from the Core perspective, the engagement has concluded. The Core’s goal is to execute a discrete high-quality experiment thus an ongoing bioinformatics consultation might not be possible without additional financial support. <strong>Closing the gap between user expectations and the typical Core product is important as is having reasonable ongoing access to expertise (internal or otherwise)</strong>. Addressing this gap involves a combination of activities involving the development of domain expertise by investigators (or their proxy) along with greater willingness by the Cores to treat engagements holistically as opposed to “one-and-done” transactions.</p>
</div>
<div id="data-access-and-collaboration" class="section level3">
<h3><span class="header-section-number">1.1.3</span> Data Access and Collaboration</h3>
<p>A fundamental problem exists in having convenient access to data as it makes its way through the analytics pipeline and in understanding what transformations have been applied (or not) to a candidate result. A solution such as RedCap is important though it can host only the more structured part of the project data whereas the associated omics data sets typically wind up on an Emory Box account that cannot be computed against. This leads to data being distributed to colleagues resulting in multiple copies that are transformed according to the needs of whomever is currently working on it (statistician, biologist, bioinformatician). Unfortunately, the transformed data and results (as well as the code used to produce it) are rarely reintegrated back into a repository for common reference.</p>
<p>This is understandable as there is no common platform upon which to perform the work which could also serve as a sharing / collaboration environment. It is generally agreed that maintaining projects on laptops is a problem though it remains a popular approach. However, not all PIs feel comfortable sharing data especially in scenarios where the data has been difficult to obtain. It’s agreed that research reproducibility is important but there are sensitivities to having data commonly available during the project. (Appropriate security approaches can help here). Independently of these concerns, the funding agencies, (via concepts such as the <a href="https://www.force11.org/group/fairgroup/fairprinciples">“FAIR Guiding Principles”</a> for data management and stewardship), are very much concerned with the “long-term care” of valuable digital assets, thus the appropriate annotation and archival of data is an obligation.</p>
</div>
<div id="the-promise-of-cloud-computing" class="section level3">
<h3><span class="header-section-number">1.1.4</span> The Promise of Cloud Computing</h3>
<p>Computation remains a challenge in that the School does not currently “own” any significant computational resources which has led researchers to leverage personal and external professional relationships to accomplish work. This approach 1) does not scale well, 2) works against the idea of reproducible research, and 3) impairs collaborative ability. Cloud computing offers unprecedented access to scaleable, on-demand computation and storage resources in a manner that allows Emory researchers to be competitive with institutions that possess large, on-premise computational infrastructure. In effect, services such as Amazon and Google democratize computing by enabling access to anyone with a credit card and a willingness to learn how to leverage the environment. However, the path to productivity is not clear and there is confusion on how to approach Amazon services even before considering how to execute bioinformatics jobs and pipelines. The language of Amazon is one of enterprise services and architecture as opposed to research computing.</p>
<p>The Emory LITS organization is in the process of rolling out a solution (ETA 2019) to help researchers but the presentations to date point to a model that involves a portal customized primarily for the benefit of LITS security interests. The larger question relates to what extent the cloud team will provide “in the trenches”, hands on training and help with selecting and managing various instance types, storage resources, and databases above and beyond the architectural level. This is where existing Cores could fill the gap or a more Nursing-centric approach could be helpful.</p>
</div>
</div>
<div id="executive-summary-and-initial-recommendations" class="section level2">
<h2><span class="header-section-number">1.2</span> Executive Summary and Initial Recommendations:</h2>
<p>Note that these represent suggested starting points which can flexibly be implemented. Most, if not all, of these recommendations are based on (or are motivated by) suggestions from faculty who have attempted to address these problems in various ways including leveraging personal, professional, and external relationships. Further discussion is warranted particularly on items relating to the formation of a central resource (1.2.2) and the licensing of a data management and computation framework (1.2.1). The following statements were derived from the more detailed sections (2-6) of this document which in turn were based on the faculty interviews.</p>
<div id="data-management" class="section level3">
<h3><span class="header-section-number">1.2.1</span> <strong>Data management</strong></h3>
<blockquote>
<p>The School should consider use of a tool such as DNANexus or Seven Bridges Genomics. either of which would provide comprehensive support for managing sequencing-based projects, analytics, as well as the convenient addition of self-developed pipelines.</p>
</blockquote>
<p>Data management is a very important aspect of analysis and research although the current practices for supporting these activities are less than ideal. A framework is needed to maintain experimental data, code, and notebooks. There are currently three storage solutions (Isilon, Box, and Amazon) at Emory and it isn’t clear to researchers when and why to use them particularly when wanting to perform computation. Intermediate results and data transformations are not maintained alongside original data which impairs reproducibility. The data residing in Emory Box cannot be computed against and must be moved to individual computers (sometimes several) for downstream analysis. <strong>Emory should provide a comparable amount of free Amazon S3 storage to that which is available in Emory Box</strong>. S3 storage can be accessed by compute instances and would thus incentivize users to adopt Amazon cloud services.</p>
<p>This would address a number of concerns of having “everything in one place” (to the extent that it is possible) along with computational results in a format that would enable reproducibility. Additionally, a solution such as DNANexus would facilitate the integration of genomic data with clinical and other phenotypic data in a secure and compliant environment. While RedCap is useful for maintaining study information, being able to link in sequencing and sample information can be challenging. For a more detailed analysis of this topic please consult section 4) Interaction with Cores.</p>
</div>
<div id="nursing-centric-analytics-omics-resource" class="section level3">
<h3><span class="header-section-number">1.2.2</span> <strong>Nursing-Centric Analytics ‘Omics’ Resource</strong></h3>
<blockquote>
<p>The formation of a nursing-centric support group is recommended. Such a group would not replace or diminish the relationships with existing University Cores but rather improve the working dynamic since Nursing investigators would have an advocate acting on their behalf</p>
</blockquote>
<p>Availability of professionals who are well-versed in analytics, statistics, and Omics are critical to the research mission of the School. <strong>To this end, the formation of a nursing-centric support group is recommended</strong>. It would provide services for analytics, computing, and interactions with sequencing facilities (where desired) and generally exist for the benefit of investigators. <strong>This entity should be independent of IT as its aims, workloads, and dynamics are quite different from the priorities of daily IT.</strong> Whether this service center is an “official university core” or a formalized group is up for debate although the feedback on this topic is clear in that the group should prioritize Nursing School interests while also being available for salary support via external projects albeit secondarily. Duties would involve assistance with data management, cleaning, preliminary analysis, and ongoing participation in projects. Organizing training and providing orientation to the framework mentioned in item #1 would also be on the menu of services.</p>
<p>It is important to note that such a resource could be referenced in applications for grants involving large scale processing. This would be evidence of a formal institutional commitment to Nursing analytics. Opinions vary as to the exact composition of this group and/or if it should draw from existing personnel (who aren’t already fully funded). Moreover, one investigator expressed concern about hosting such a group internally as resources might then be arbitrarily redirected to non-research projects or uniquely to projects associated with the most senior faculty. However, it was generally agreed that having more domain specific expertise specifically for the School of Nursing is definitely needed.</p>
<p>In terms of implementing this resource, it is important to note that research computation presents a set of problems related to though significantly different from the production difficulties associated with Nursing Information Technology. While some possible merger of the two units could be imagined, it should be staged in over time to give the Nursing Omics resource a reasonable chance of success with the added baggage of solving existing problems with Nursing IT.</p>
</div>
<div id="improved-project-logistics" class="section level3">
<h3><span class="header-section-number">1.2.3</span> <strong>Improved Project Logistics</strong></h3>
<blockquote>
<p>Faculty want access to personnel familiar with the statistics, computation, and bioinformatics as part of the research planning phase. This could be part of the previously mentioned Nursing Omics Resource or independent of it.</p>
</blockquote>
<p>Researchers would like more detailed upfront estimates of statistics, analytics, and computation costs to better project cost of staff involvement. Not knowing how much (in time, money, and personnel) the key aspects of a project will “really” involve is a concern. This could be implemented separately of the proposed local Nursing Omics resource though if such a resource did exist then this person could also be part of that that. The overall idea is to have accurate and reliable estimates of sequencing, analytics, and computation work. Some junior faculty feel that they cannot fully access personnel due to lack of funding or because the more experienced personnel are already mostly engaged on other projects. In general, having a standard logistical workflow that specifies who to involve, when, and to what extent is needed. This includes collaborating domain expertise (statistics and bioinformatics personnel) as well as key Core facilities - internal or external to Emory.</p>
</div>
<div id="research-desktop-support" class="section level3">
<h3><span class="header-section-number">1.2.4</span> <strong>Research Desktop Support</strong></h3>
<blockquote>
<p>Nursing School resesrchers want a higher degree of “research aware” desktop support services from Nursing IT.</p>
</blockquote>
<p>While learning more about Amazon is important, all of the faculty interviewed have expressed disappointment with current School IT support which, as a group, does not seem motivated or interested in helping staff, students, or faculty particularly as it relates to research. Nursing School researchers need desktop assistance particularly as it relates to installing, updating, and running opensource packages such as R, Python, Anaconda and associated opensource tools. The School faculty should not expect IT personnel to actively conduct research or understand the intricacies of software workflows. However, the IT group should be able to troubleshoot package installation conflicts, prerequisites, and perform upgrades necessary to enable base level research exploration. In general most faculty believe that Nursing IT should be incentivized to more aggressively support research including a basic level of cloud orientation. The fact that the IT group is currently not oriented towards such activities should be not an ongoing rationale to keep the status quo.</p>
</div>
<div id="high-performance-computing" class="section level3">
<h3><span class="header-section-number">1.2.5</span> <strong>High Performance Computing</strong></h3>
<blockquote>
<p>The School should identify paths to cloud and command-line literacy via courses and training developed internally and in conjunction with Amazon Web Services.</p>
</blockquote>
<p>Research is increasingly reliant upon both desktop and high performance computation thus a commensurate level of support is needed. Computation remains a challenge in that the School does not currently “own” any significant computational resources which has led researchers to leverage personal and external professional relationships to accomplish work. This does not scale at all. Facility with cloud computing is a challenge even for those who are computationally aware thus for newcomers it is especially so. It is not currently clear to what extent LITS or existing Cores will help researchers address the skills gap or is it expected for investigators to self-educate and recruit students to handle computational aspects of their projects. Independently of from whom the help comes, the faculty were quite clear in their interest and desire for more assistance and training.</p>
<p>If the School does <strong>NOT</strong> pursue a Nursing Centric resource then cloud education will be a function of informal resources and/or funded personnel specific to individual projects vis-a-vis a percent effort model. However, this does little to raise the computational awareness at the School level. It would be useful for the LITS cloud team or Core facilities to offer ongoing orientation and support for adopting cloud resources along with expertise in system administration, package installation, database loading (both noSQL and SQL), programming (R, Python), and other services common to biomedical and bioinformatics projects. This includes agile and flexible support for opensource tools above and beyond standard enterprise tools. In absence of such help, the School of Nursing will have to develop internal resources which of course might even be preferable. In any case, the knowledge gap must be addressed.</p>
<p>A secondary approach to the computation issue might involve partnering with the Department of Biomedical Informatics which maintains a sophisticated, well-functioning, well-supported computational cluster dedicated to aggressive research computing. This would have to be arranged but none of the projects currently being considered within the School of Nursing would represent a challenge to that environment. BMI would be an appropriate partner because research computing is “baked-in” to their mission and is thus seen as an essential service for researchers. I discussed a potential relationship with Jim Kinney (BMI System Lead) earlier this year but would need to renew those talks.</p>
</div>
<div id="alternatives-to-local-core-processing" class="section level3">
<h3><span class="header-section-number">1.2.6</span> <strong>Alternatives to Local Core Processing</strong></h3>
<blockquote>
<p>The School should develop relationships with trusted external scientific resource providers to process urgent requests for sample processing and analysis consultation.</p>
</blockquote>
<p>This should also include training resources which could be accomplished virtually, on-premise (for larger groups), or remotely. Even if all concerns with local Cores are fully addressed there remains a need for expedited processing and domain specific expertise to address backlogged projects or efforts requiring special care during processing. Not every project will require this but there should be a reliable outlet for “blocked” projects or those requiring a rapid turnaround. In particular, we should organize training to learn “best practices” in the processing and analysis of Microbiome and Metabolomics. At least one faculty member has used an external lab for Microbiome and another is contemplating the re-sequencing of a large number of samples via the Ravel Lab at the Institute of Genome Sciences to obtain domain expertise starting at the sequencing level.</p>
</div>
<div id="software-literacy" class="section level3">
<h3><span class="header-section-number">1.2.7</span> <strong>Software Literacy</strong></h3>
<blockquote>
<p>Improving general <strong>software literacy</strong> is very important as is being able to import/transform data, accomplish analytic tasks, create plots, query databases, and create digital assets.</p>
</blockquote>
<p>Many faculty pursue self-education but are interested in becoming much more self-sufficient with open source tools and the UNIX command line. This can be accomplished via participation in the Big Data course or more specific classes. A larger issue relates to what extent the Cores and LITS are interested in educating users, respectively, on domain specific pipelines and cloud computing ? Or, is it incumbent on the investigator to develop a team to ensure productivity ? Amazon presents a steep learning curve as can understanding results being returned from Cores. Some level of shared responsibility must be assumed in the development of final results. Part of the research process does involve self-education and a willingness to take a deep dive when attempting to adapt literature-based methods to local projects. Software literacy helps provide a common language and vocabulary for research.</p>
<p>One excellent (and cheap) resource is to arrange for an onsite <a href="https://software-carpentry.org/">Software Carpentry</a> session which is a 1 to 3 day workshop devoted to teaching basic lab skills for research computing. These sessions are professionally taught and include hands on labs to learn the UNIX command line, R and Python Programming, Git, SQL and Databases. These are targeted to the novice but would also serve to reinforce skills. The material is open source and maintained on GitHub so we could possibly offer the course material with local teaching resources. The following graph illustrates the motivation levels for various software and command line topics after taking a Core level Software Carpentry class.</p>
<div class="figure">
<img src="PICS/carpentry.png" width="550" />

</div>
</div>
<div id="student-recruitment" class="section level3">
<h3><span class="header-section-number">1.2.8</span> <strong>Student Recruitment</strong></h3>
<blockquote>
<p>Deliberate efforts should be made to recruit graduate students with ability and/or interest in data manipulation and analysis since the nature of research within the school will require such a background.</p>
</blockquote>
<p>It is difficult to imagine a scenario wherein a graduate student could be successful without acquiring at least a basic fluency with these tools. Obviously, such skill can be developed over time though biomedical research assumes that students can clean, reshape, and transform data both prior to analysis and afterwards. While no one believes that using <strong>only</strong> student “labor” to accomplish projects is reasonable, there is a goal of having software literate students who can “dive in” to projects. However, the School Dean did express a specific concern that it can be challenging for doctoral students who sometimes receive conflicting advice on how to structure omics-based thesis projects. So while having software savvy students is a plus, they will still require guidance in the selection and execution of their thesis work which in turn assumes the existence of a knowledgeable supportive community. For better or worse, the domain of biomedical, bioinformatics, and nursing research requires a base level of quantitative skill and facility with software packages such as R, Python, and various open source tools.</p>
</div>
</div>
<div id="the-interview-process" class="section level2">
<h2><span class="header-section-number">1.3</span> The Interview Process</h2>
<p>Conducting these interviews was a smooth process as those contacted were very forthcoming with helpful responses that were very consistent across the topic areas described in this report. Deviations of opinions did sometimes occur along the lines of junior and senior faculty although all of the senior faculty with whom I spoke were keenly aware of the difficulties facing junior faculty and appeared to be sympathetic. Everyone was collegial and supplied viewpoints that will benefit the School as a whole. The concerns expressed were entirely relevant to improving research processes and it’s clear that even though there are some challenges, people are nonetheless working very hard to get things done. The School of Nursing has definite strengths in its clinical programs combined with accomplished and talented faculty. It remains a priority to continue to apply for data-based grants and to this end the School must be able to point to official and strong support for analytics and computation within proposals to be competitive.</p>
<p>The participants in the interview process were: Linda McCauley, Anne Dunlop, Carolyn Clevenger, Jessica Wells, Irene Yang, Rebecca Mitchell, Jinbing Bai, Erin Ferratti, Betsy Corwin, Nicole Carslon, Despina Tsementzi, Anna Knight, Deborah Bruner, Mary Gullate, Melinda Higgins, and Jason Atkins. I also had a discussion with Karl Moran at Microbiome Insights since they performed work for Irene Yang. I also contacted Ethenia Why at the Biostatistics Core at Hopkins for comments on their role in Microbiome analysis. I’ve also got an inquiry to Steven Gill at the University of Rochester Microbiome Core. I’ve had conversations with Madhusmita Behera who is the Director of Informatics at Winship Cancer Institute which is also working towards a usable solution for Data Management and Computation.</p>
<p>The following sections (2-6) represent the interview summary information which was used as a basis for the above recommendations. Each section takes a deeper dive into the area under consideration.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="data-management-1.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": ["Nursing Report.pdf", "Nursing Report.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
